fusillade/api/users/__init__.py

```diff
@@ -40,6 +42,17 @@ def put_user(token_info: dict, user_id: str):
     return resp
 
 
+@authorize(['fus:GetUser'], ['arn:hca:fus:*:*:user/{user_id}/owns'], ['user_id'])
+def get_users_owns(token_info: dict, user_id: str):
+    next_token, per_page = get_next_token(request.args)
+    user = User(directory, user_id)
+    return get_page(user.get_owned,
```

Comment:
lol, must have missed this the first time.

Reply:
Hopefully no one takes it the wrong way

Reply:
Ha, I would hope not.